### PR TITLE
[INLONG-4711][Manager] Fix JSON parse failed in client

### LINKED ISSUE #4711: [Improve][Manager] Fix json parse failed in Manager Client
### Description

_No response_

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by kipshi**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/incubator-inlong/issues) number)*

- Fixes #4711 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

**Comment by dockerzhang** [comment]
@kipshi UT failed, please check again.

**Review by dockerzhang — APPROVED** [review]

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/InnerInlongManagerClient.java [hunk 1]
```diff
@@ -144,12 +149,21 @@ public Boolean isGroupExists(String inlongGroupId) {
...
+    @SneakyThrows
     public InlongGroupInfo getGroupInfo(String inlongGroupId) {
         AssertUtils.notEmpty(inlongGroupId, "InlongGroupId should not be empty");
 
-        Response<InlongGroupInfo> responseBody = executeHttpCall(inlongGroupApi.getGroupInfo(inlongGroupId));
+        Response<Object> responseBody = executeHttpCall(inlongGroupApi.getGroupInfo(inlongGroupId));
         if (responseBody.isSuccess()) {
-            return responseBody.getData();
+            JSONObject groupInfoJson = JsonUtils.parseObject(
+                    JsonUtils.toJsonString(JsonUtils.toJsonString(responseBody.getData())),
```
Comment (healchow) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/InnerInlongManagerClient.java:159]: Why call `toJsonString` 2 times here?

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongGroupImpl.java [hunk 2]
```diff
@@ -102,8 +107,19 @@ public InlongGroupContext init() throws Exception {
                         processView.getStatus()));
 
         // init must be NewGroupProcessForm
+        // compile with old cluster
+        JSONObject formDataJson =
+                JsonUtils.parseObject(JsonUtils.toJsonString(JsonUtils.toJsonString(processView.getFormData())),
```
Comment (leosanqing) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongGroupImpl.java:112]: Why call toJsonString 2 times here?

Comment (kipshi) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongGroupImpl.java:112]: same as above

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/InnerInlongManagerClient.java (hunk 1 above)
Comment (kipshi) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/InnerInlongManagerClient.java:159]: body is hashMap
first ( toJsonString ) make it string
second( toJsonString ) make it json_string

**Review by leosanqing — APPROVED** [review]

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/InnerInlongManagerClient.java (hunk 1 above)
Comment (healchow) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/InnerInlongManagerClient.java:159]: See this:

2be5da2d56a97360df8d0608dc430e50

**Pull request merged**